.github/workflows/periodic-merge-24h.yml

```diff
@@ -32,6 +32,10 @@ jobs:
             into: staging-next-21.05
           - from: staging-next-21.05
             into: staging-21.05
+          - from: release-21.11
+            into: staging-next-21.11
+          - from: staging-next-21.11
+            into: staging-21.11
```

Comment:
Maybe 6h would be more appropriate until 21.11 has been released? Can imagine that staging and release branch will be higher velocity for a period?

Reply:
The hydra job for staging-next-21.05 is currently on a two day cycle, so I think daily is fine, unless people force evals more often than that.